app/controllers/events_controller.rb

```diff
@@ -4,10 +4,10 @@ class EventsController < ApplicationController
   before_action :is_logged_in?, only: [:student, :coach]
 
   def index
-    events = [ Workshop.past.all ]
-    events << Course.past.all
-    events << Meeting.past.all
-    events << Event.past.all
+    events = [ Workshop.recent.all ]
+    events << Course.recent.all
```

Comment:
on further reflection we can just merge this PR in and address the course issue in another PR. It is peppered around the code but is unused, might be good to remove every reference to it except the Model etc. in case we need it in the future.